pallene/driver.lua

```diff
@@ -125,6 +126,18 @@ end
 --    compile("c", "so", "foo.c)      --> outputs "foo.so"
 --
 
+function driver.translate(input_file_name, input)
```

Comment:
I think it is a bit strange that the test suite is using different code paths for the cases that give errors and the cases that don't. In the former we invoke pallenec and in the latter we call a special function from driver.lua.

What do you think about calling pallenec in both cases?

```
local function compile(pallene_code)
    assert(util.set_file_contents("__test__.pln", pallene_code))
    local ok, _, _, error_message = util.outputs_of_execute("./pallenec __test__.pln --emit-lua")
    return ok, error_message
end

local function assert_translation(pallene_code, expected)
    assert(compile(pallene_code))
    local contents = util.get_file_contents("__test__.lua")
    assert.are.same(expected, contents)
end

local function assert_translation_error(pallene_code, expected)
    local ok, err = compile("__test__.pln", pallene_code)
    assert.is.falsy(ok)
    assert.match(expected, err, 1, true)
end
```